app/workers/activitypub/delivery_worker.rb

```diff
@@ -23,9 +23,10 @@ class ActivityPub::DeliveryWorker
   HEADERS = { 'Content-Type' => 'application/activity+json' }.freeze
 
   def perform(json, source_account_id, inbox_url, options = {})
-    return unless DeliveryFailureTracker.available?(inbox_url)
-
     @options        = options.with_indifferent_access
+
+    return unless DeliveryFailureTracker.available?(inbox_url) || @options[:bypass_availability]
```

Comment:
I'd check the option before the `DeliveryFailureTracker`, as this is less expensive. But this is very minor either way.

```suggestion
    return unless @options[:bypass_availability] || DeliveryFailureTracker.available?(inbox_url)
```